docs/general/naming.md

```diff
@@ -258,15 +294,15 @@ For example if we have a namespace `system.process` which contains all metrics r
 to the processes then to represent the count of the processes we can have a metric named
 `system.process.count`.
 
-#### Do Not Use `total`
+#### Do not use `total`
 
 UpDownCounters SHOULD NOT use `_total` because then they will look like
 monotonic sums.
 
 Counters SHOULD NOT append `_total` either because then their meaning will
 be confusing in delta backends.
 
-### Instrument Naming
+### Instrument naming
```

Comment:
Why are we changing title convention everywhere?

Reply:
it's what otel.io does for link titles - https://github.com/open-telemetry/semantic-conventions/issues/169, pretty much all headings we have are in Sentence case now.